src/react/src/__tests__/components/DashboardFacilityCard.test.js

```diff
@@ -0,0 +1,256 @@
+                        "updated_at": "2024-08-02T14:30:18.426821Z",
+                        "contributor_name": "JD Williams and Company LTD trading as N Brown ",
+                        "contributor_id": 1610,
+                        "value_count": 4,
+                        "is_from_claim": false,
+                        "field_name": "product_type",
+                        "verified_count": 0
+                    },],
+                    "parent_company": [
+                        {
+                        "id": 262674,
+                        "is_verified": false,
+                        "value": {
+                            "name": "Noman Group",
+                            "raw_value": "Noman Group"
+                        },
+                        "updated_at": "2022-11-14T00:45:27.294312Z",
+                        "contributor_name": "Zaber and Zubair Fabrics Ltd",
+                        "contributor_id": 3922,
+                        "value_count": 2,
+                        "is_from_claim": true,
+                        "field_name": "parent_company",
+                        "verified_count": 0
+                    },]
+                },
+                "created_from": {
+                    "created_at": "2020-03-06T09:31:31.443253Z",
+                    "contributor": "a Multi-Stakeholder Initiative"
+                },
+                "sector": [                    {
+                    "updated_at": "2024-08-18T00:46:41.082596Z",
+                    "contributor_id": 10104,
+                    "contributor_name": "Marks & Spencer",
+                    "values": [
+                        "Home"
+                    ],
+                    "is_from_claim": false
+                },]
+            }
+        },
+        fetching: false,
+        error: null,
+        handleEnterKeyPress: jest.fn(),
+        title: "Target facility for merge",
+        highlightBackground: false,
+    };
+
+    const renderComponent = (props = {}) =>
+        renderWithProviders(
+            <DashboardFacilityCard {...{...defaultProps, ...props}}/>
+        );
+
+    it('renders without crashing', () => {
+        renderComponent();
+    });
+
+    it('renders the facility title', () => {
+        const { getByText } = renderComponent();
+
+        expect(getByText("Target facility for merge")).toBeInTheDocument();
+    });
+
+
+    it('renders the NOT claimed facility card without green background', () => {
+        const { container } = renderComponent();
+
+        expect(container.firstChild).not.toHaveStyle(`background:${COLOURS.GREEN}`)
+    });
+
+    it('renders the claimed facility card with green background', () => {
+        const { container } = renderComponent({highlightBackground: true,
+            data: {
+                ...defaultProps.data,
+                properties: {
+                    ...defaultProps.data.properties,
+                    is_claimed: true,
+                },
+            },});
```

Comment:
I believe that moving the complex prop object into a variable can improve the readability and maintainability of the test.

    const claimedFacilityProps = {
        highlightBackground: true,
        data: {
            ...defaultProps.data,
            properties: {
                ...defaultProps.data.properties,
                is_claimed: true,
            },
        },
    };

    const { container } = renderComponent(claimedFacilityProps);